jetbrains-core/tst/software/aws/toolkits/jetbrains/services/lambda/python/PythonLambdaHandlerResolverTest.kt

```diff
@@ -180,6 +242,22 @@ class PythonLambdaHandlerResolverTest {
         }
     }
 
+    private fun assertHandlerDetermineHandlers(handler: String, shouldBeFound: Boolean) {
+        runInEdtAndWait {
+            val elementSet = findHandler(handler)
+            assertThat(elementSet).hasSize(1)
```

Comment:
Confused a bit on why this will always be found, if shouldBeFound is false

Reply:
Let me change the name. It should be found as a valid candidate handler but then be filtered by determineHandler.